<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorBalanceDialog</class>
 <widget class="QDialog" name="ColorBalanceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Баланс цветов</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="redBalanceL">
       <property name="text">
        <string>Голубой-Красный:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="redBalanceSlider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="redBalanceSB">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="greenBalanceL">
       <property name="text">
        <string>Пурпурный–Зелёный:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="greenBalanceSlider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="greenBalanceSB">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="blueBalanceL">
       <property name="text">
        <string>Жёлтый–Синий:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="blueBalanceSlider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="blueBalanceSB">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="text">
        <string>Да</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>greenBalanceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenBalanceSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenBalanceSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenBalanceSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redBalanceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redBalanceSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redBalanceSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redBalanceSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueBalanceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueBalanceSB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueBalanceSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueBalanceSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okPB</sender>
   <signal>clicked()</signal>
   <receiver>ColorBalanceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPB</sender>
   <signal>clicked()</signal>
   <receiver>ColorBalanceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redBalanceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ColorBalanceDialog</receiver>
   <slot>onRedBalanceChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenBalanceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ColorBalanceDialog</receiver>
   <slot>onGreenBalanceChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueBalanceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ColorBalanceDialog</receiver>
   <slot>onBlueBalanceChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onRedBalanceChanged(int)</slot>
  <slot>onGreenBalanceChanged(int)</slot>
  <slot>onBlueBalanceChanged(int)</slot>
 </slots>
</ui>
